api/sourceregistrations/views.py

```diff
@@ -42,12 +43,3 @@ class ProviderRegistrationViewSet(viewsets.ModelViewSet):
 
     def get_queryset(self):
         return ProviderRegistration.objects.filter(submitted_by_id=self.request.user.pk)
-
-    def create(self, request, *args, **kwargs):
```

Comment:
Should this be deleted?

Reply:
Yeah, it doesn't do anything that the default `create` doesn't do